libraries/c_sdk/standard/serializer/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ afr_module_sources(
     INTERFACE
         "${test_dir}/iot_tests_serializer_cbor.c"
         "${test_dir}/iot_tests_serializer_json.c"
+	"${test_dir}/iot_tests_deserializer_json.c"
```

Comment:
Whenever we add a new file, we are supposed to update all project files (test project in your case). You don't need to make the change in this PR. I just raise this concern.

Reply:
@qiutongs I updated the PR with the revised project files.

Reply:
Looks good